lib/WebServer/MiLightHttpServer.cpp

```diff
@@ -128,7 +128,19 @@ void MiLightHttpServer::handleSystemPost() {
       delay(100);
 
       ESP.restart();
-    }
+
+      handled = true;
+    } else if (request["command"] == "reset_wifi") {
```

Comment:
I wonder if a name like `reset_wifi_config` might be clearer? `reset_wifi` makes it sound like it might just be re-connecting to the AP.